docs/about/history.md

```diff
@@ -0,0 +1,190 @@
+involvement and contributions”. This was described in the projects
+[Governance][governance-2020] page which was the inspiration for the current
+[Valkey governance](valkey-governance).
+
+[governance-2020]: https://web.archive.org/web/20200709170526/https://redis.io/topics/governance
+[valkey-governance]: https://github.com/valkey-io/valkey/blob/unstable/GOVERNANCE.md
+
+In 2021, the Redis Labs changed its name to Redis.
+
+The end of open source Redis
+----------------------------
+
+Despite the seemingly community-based core team, Redis (the company) maintained
+a majority in the core team as well as ownership of the Redis trademark. In
+2024, the company changed the license of Redis from the open source BSD license
+to dual source-available licenses. This was announced in a blog post [Redis
+Adopts Dual Source-Available Licensing][redis-source-avail] and the license
+change was [commited to the repository][change-license-pr] the same day.
+
+[redis-source-avail]: https://redis.io/blog/redis-adopts-dual-source-available-licensing/
+[change-license-pr]: https://github.com/redis/redis/pull/13157
+
+Neither of the licenses Redis Source Available License (RSALv2) nor the Server
+Side Public License (SSPLv1) are open source licenses, as neither meet the
+criteria of an open source license.
+
+RSALv2 forbids the use of the software in database products, to prevent their
+competitors from providing database products and services. Such a restriction is
+not in line with The Open Source Definition, criterion #6 “The license must not
+restrict anyone from making use of the program in a specific field of endeavor”
+by the Open Source Initiative, nor “the freedom to run the program as you wish,
+for any purpose (freedom 0)” in the Free Software Foundation's definition of
+Free Software. The SSPL has similar restrictions, explained in the [SSPL article
+on Wikipedia][sspl].
+
+[sspl]: https://en.wikipedia.org/wiki/Server_Side_Public_License
+
+The birth of Valkey
+-------------------
+
+The many contributors, not least from companies providing hosted Redis-derived
+or Redis-compatible databases services as a service, just like Redis (the
+company) does, and which have been using and contributing to Redis (the open
+source software) just as long as Redis (the company) has existed, it was an easy
+decision by many contributors to continue the open source development as usual
+under the BSD license.
+
+As Redis (the company) owns the trademark rights to the name Redis, the open
+source project needed to continue under a different name. A group of six active
+contributors (one from each of Alibaba, Amazon, Ericsson, Google, Huawei and
+Tencent) with the support from several other companies launced Valkey as a Linux
+Foundation project. It was announced in a [press release][lf-launch] only eight
+days after Redis' license change.
+
+[lf-launch]: https://www.linuxfoundation.org/press/linux-foundation-launches-open-source-valkey-community
```

Comment:
I would include the second lf announcement, https://www.linuxfoundation.org/press/valkey-community-announces-release-candidate-amid-growing-support-for-open-source-data-store, which included the support of 7 more companies as well as the announcement for 7.2.5